config.js

```diff
@@ -1,8 +1,8 @@
 var config = {}
 
-config.endpoint = "GRAPHENDPOINT.gremlin.cosmosdb.azure.com";
+config.endpoint = "wss:/GRAPHENDPOINT.gremlin.cosmosdb.azure.com:443/gremlin";

```

Comment:
nit: it's best to just declare all of these in the config at instantiation. It's slower to do it like you've done (though it doesn't really matter here since it's a one time cost/etc., hence nit)